pages/create-account.js

```diff
@@ -0,0 +1,138 @@
+import Head from 'next/head';
+import React, { useState } from 'react';
+import styles from '../styles/Home.module.css';
+
+export default function Account() {
+  const [inputErrorMessages, setErrorMessages] = useState({
+    firstName: '',
+    lastName: '',
+    username: '',
+    password: '',
+    confirmPassword: '',
+  });
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+
+    const inputs = {};
+    const formData = new FormData(e.target);
+    formData.forEach((value, key) => (inputs[key] = value));
```

Comment:
Is there a reason why you're copying the `FormData` to an object? Can't we extract what we need from the `FormData` directly? See https://developer.mozilla.org/en-US/docs/Web/API/FormData/get

Reply:
Originally thought it would be easier to send a JSON object, but looks like thats what FormData objects are used for. Removed redundant input variable.

Reply:
Originally thought it would be easier to send a JSON object, but looks like thats what FormData objects are used for. Removed redundant input variable.